src/cloud/index.ts

```diff
@@ -332,7 +336,8 @@ export class Cloud extends APICarrier {
 
   public async makeBaseResourceListRequest<T, TL, K, O, E>(proto: BaseResourceCreateable<T, K>, listProto: BaseResourceList<TL>, errorProto: BaseResourceErrorProto<E>, options?: BaseResourceEntityFetchOptions<O>): Promise<T[] | K[] | undefined> {
     try {
-      const res = await this.http.getClient().get(`${this.cloudBase}/${listProto.endpoint}`, {
+      const endpoint = options?.endpoint ?? listProto.endpoint
+      const res = await this.http.getClient().get(`${this.cloudBase}/${endpoint}`, {
```

Comment:
In the current implementation, it's only possible to have a static endpoint. I'm adding the possibility to add a dynamic endpoint i.g. `api/v0/interverse/organizations/${this.organizationId}/keys`